OutOfSchool/OutOfSchool.WebApi.Tests/Services/StatusServiceTests.cs

```diff
@@ -163,15 +175,10 @@ public async Task Delete_WhenIdIsInvalid_ThrowsArgumentOutOfRangeException()
 
     private void SeedDatabase()
     {
-        using var context = new OutOfSchoolDbContext(options);
-        {
-            context.Database.EnsureDeleted();
-            context.Database.EnsureCreated();
-
-            var institutionStatuses = InstitutionStatusGenerator.Generate(5);
-            context.InstitutionStatuses.AddRange(institutionStatuses);
+        context.Database.EnsureDeleted();
+        context.Database.EnsureCreated();
 
-            context.SaveChanges();
-        }
+        // New version of InMemoryTestDB uses ModelBuilder to initialize data from common OutOfSchool contex.
+        // Don't need to use seeding of elements.
```

Comment:
(Commenting standard functionality) Thanks for the sharing this info, at same time could you reconsider the place? 
For me, it looks that this is a standard functionality, not a "hack" or tricky issue, so we don't need to put standard documentation into our code. 
Moreover, there are `SeedDatabase()` in other test classes, and it will be hard (and inefficient) to add the same comment everywhere.

Usually I suggest to put such knowledge sharing notes about important code changes in the PR comment, and also mention it somewhere in team chat/meetings.

Reply:
Ok. It's the new behavior for InMemoryDatabase. I didn't seed this DB. I tried to describe why.

Reply:
Cleared.